### DISCUSSION TITLE
Use M3/M4 Shortcut Key to toggle Multizone

### DISCUSSION THREAD

**Discussion by justing6** [seerge/g-helper#310] [Ideas]
Hi, apologies if this has been suggested before but I couldn't find anything. It would be great to be able to bind M3 or M4 to toggle the Multizone setting for laptops with miniled displays! 

Asus' miniled implementation is great for videos and games and for increasing brightness for outdoor usage, but pretty terrible for anything else. I really only toggle it on for games or videos (and of course HDR where its forced on). The button on G-Helper is already a lot better than going into armory crate, but it would be even nicer if I could put it on M3!

**Comment by seerge** [comment]
@JustinGuiao hello, you can check this build - i have added an option for M3/M4 called "Toggle Miniled" under Extra

[GHelper.zip](https://github.com/seerge/g-helper/files/11381039/GHelper.zip)

**Comment by justing6** [comment]
Works perfectly, thank you!

**Comment by turtleshell64** [comment]
So glad you implemented this before armory crate did, however I notice only the m4 key has the "open ghelper window" function now and all other keys don't have it. Was that removed on purpose? Also been curious where the settings get saved to since I don't see any ini files or whatever where my ghelper exe is

**Reply by seerge** [reply]
It's on m4 only by design (from the beginning), from what button do you want to launch it instead? :) settings are saved in app config under %appdata%/ghelper as readme says

**Reply by turtleshell64** [reply]
> It's on m4 only by design (from the beginning), from what button do you want to launch it instead? :) settings are saved in app config under %appdata%/ghelper as readme says

Ah, previous versions let me bind open ghelper window to any of the 3 shortcuts. I don't mind using m4 but whenever I hit that, armory crate pops up a window and bugs me to go to the website and download the program (unless you know how to turn that off). Maybe there's a way to have ghelper suppress the armory crate notification?

**Reply by seerge** [reply]
That’s pretty much first item in FAQ https://github.com/seerge/g-helper#how-do-i-stop-the-armory-crate-install-popup-appearing-every-time-i-press-the-m4--rog-key :)

**Reply by turtleshell64** [reply]
> That’s pretty much first item in FAQ https://github.com/seerge/g-helper#how-do-i-stop-the-armory-crate-install-popup-appearing-every-time-i-press-the-m4--rog-key :)

Awesome, that's my bad for not fully reading. Seems I had to disable from bios before I could delete the exe. I guess all is well now since m4 no longer bugs me! Many thanks!
